### ISSUE TITLE
QuickPick: group name should be part of first element aria label

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141856]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #Narrator; #NVDA; #ScreenReader; #WCAG4.2.5 #AINotInScope; #FTP; #STP;

### Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

### Repro Steps:
1)Turn on Narrator/NVDA.
2)Open Visual studio code editor.
3)Open the command Pallet by pressing "F1" key and type "Theme"
4)Using Down arrow key select "Preference color Theme" and press enter key to select.
5)Using "Up/Down" arrow key select different available theme.
6)verify that screen reader is announcing the associated label for the 'recently used' and 'Other commands' or not.

### Actual Result:
Screen reader does not announce associated label for the 'recently used' and 'Other commands'..

### Expected Result:
Screen reader should announce the associated label for the 'recently used' and 'Other commands'.

### User Impact:
Screen Reader users will not get the complete information about the controls.
![A11y_VisualStudioCodeClient_customization and setting_screen Reader](https://user-images.githubusercontent.com/98530069/151744636-340047cb-0f16-4ff7-8001-22e6bcc35cb2.gif)

**Comment by isidorn** [comment]
The issue here is that the group label "recently used" does not get read.
@TylerLeonhardt I suggest to append the group name to the aria label of the first element in the quick pick that belongs to that group. I think it should be a low hanging fix.
